assets/src/web-stories-block/css/views/circles.css

```diff
@@ -1,20 +1,16 @@
 /* Web Stories: Circles View */
-.is-view-type-circles {
+.is-view-type-circles .web-stories-list__inner-wrapper {
```

Comment:
Where is `web-stories-list__inner-wrapper` used now?

Reply:
Following is the new structure
- .wp-stories-list ( main div )
  - .wp-stories-list__inner-wrapper ( immediate first child, this will have our stories in it )

You can check the new HTML on [Google Demo site](https://google-webstories.rt.gw/?noamp=available) for both AMP and non-AMP.